index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Natasha Ivanenko - CV</title>
+</head>
+<style>
+    body {width: 600px; font-family: 'Times New Roman', Times, serif;}
+    h1 {margin-left: 64px; margin-top: 48px;}
+    h2 {margin-left: 64px; margin-top: 12px; margin-bottom: 24px;}
+    h3 {margin-left: 64px; margin-top: 24px;}
+    h4 {margin-left: 64px; margin-top: 1px; margin-bottom: 8;}
+    p {margin-left: 64px;}
+    a {margin-left: 64px; margin-top: 12px;}
+</style>
```

Comment:
Стили или удали или перенеси внутрь тега `head`

```suggestion
    <style>
        body {width: 600px; font-family: 'Times New Roman', Times, serif;}
        h1 {margin-left: 64px; margin-top: 48px;}
        h2 {margin-left: 64px; margin-top: 12px; margin-bottom: 24px;}
        h3 {margin-left: 64px; margin-top: 24px;}
        h4 {margin-left: 64px; margin-top: 1px; margin-bottom: 8;}
        p {margin-left: 64px;}
        a {margin-left: 64px; margin-top: 12px;}
    </style>
</head>
```